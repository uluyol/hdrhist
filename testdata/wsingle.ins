0
1
123
1455
232
90999
---
