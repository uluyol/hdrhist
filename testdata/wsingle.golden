0.000,0.001,0.091,HISTFAAAACp42pNpmSzMwMDAwwABzFCaEUL+/2//ASLwlZHpOiNTtzDTzngmAJ6tCBE=
